core/src/main/java/org/apache/iceberg/variants/Variants.java

```diff
@@ -209,59 +213,59 @@ public static VariantPrimitive<Void> ofNull() {
     return new PrimitiveWrapper<>(PhysicalType.NULL, null);
   }
 
-  static VariantPrimitive<Boolean> of(boolean value) {
+  public static VariantPrimitive<Boolean> of(boolean value) {
```

Comment:
Needed to expose these to create test values in the Parquet package.

Reply:
I think it would be fine to expose it here, but it we could also add some builders in the test module if we want to not have as much exposure. So TestParquet depends on TestCore and core has public builders for Variants.

Reply:
I think that we're going to want these in order to be able to construct variants anyway, so I'm inclined to leave them as public, at least in core.